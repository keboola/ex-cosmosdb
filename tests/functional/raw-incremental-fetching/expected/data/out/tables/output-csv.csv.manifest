{
    "incremental": false,
    "columns": [
        "id",
        "data"
    ],
    "primary_key": [
        "id"
    ],
    "column_metadata": {
        "id": [
            {
                "key": "KBC.datatype.nullable",
                "value": false
            }
        ],
        "data": [
            {
                "key": "KBC.datatype.nullable",
                "value": true
            }
        ]
    }
}