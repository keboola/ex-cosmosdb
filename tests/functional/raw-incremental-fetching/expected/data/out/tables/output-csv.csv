"1","{""id"":""1"",""name"":""Erika Harrell"",""age"":18}"
"2","{""id"":""2"",""name"":""Nahla Nunez"",""age"":20}"
"3","{""id"":""3"",""name"":""Jaeden Escobar"",""age"":14}"
"4","{""id"":""4"",""name"":""Zane Craft"",""age"":13}"
"5","{""id"":""5"",""name"":""Sebastien Mora"",""age"":32}"
